content/chainguard/chainguard-images/how-to-use/verifying-chainguard-images-and-metadata-signatures-with-cosign/index.md

```diff
@@ -49,35 +86,41 @@ cosign verify \
 ### Private/Dedicated Registry
 
```

Comment:
Probably worth making clear that they will need to replace IMAGE with an IMAGE they are entitled to.